src/modules/wreck/job.c

```diff
@@ -399,6 +400,7 @@ static void job_create_cb (flux_t *h, flux_msg_handler_t *w,
                                           "ntasks", &job->ntasks,
                                           "nnodes", &job->nnodes,
                                           "ncores", &job->ncores,
+                                          "ngpus", &job->ngpus,
```

Comment:
Extra `s?i` parameter (or `s?:i` for consistency) is required for the `"ngpus", &job->ngpus` pair in the format string for  `flux_request_unpack()` here as above

Reply:
Heh.. I think this explains weird sched failure I was getting yesterday. 

I had to ask though. Why `s?`  not `s` ?

Reply:
In the past these request messages only included members that were actually set by the user. If not set they were assumed zero.

It may make sense now to have these members of this message be required.